### PR TITLE
Add nav examples to guides

### PR DESCRIPTION
**Pull request by bsudekum**
Pulls examples from https://github.com/mapbox/navigation-ios-examples and puts them in the guides folder. This will get published to 

image

The update script could be added the documentation script if we'd like or it can be ran independently. 

/cc @mapbox/navigation-ios

### CODE REVIEW

**Comment by bsudekum** [comment]
Let's rename `Guides` to `Examples`.

File: docs/guides/Storyboards.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
-# Setting up Mapbox Navigation in a storyboard
```
Comment (1ec5) [inline docs/guides/Storyboards.md]: Looks like this PR deletes the storyboard guide, yet it remains in the table of contents under “Examples”. We should keep the storyboard guide as a guide – that is, under docs/guides/ and the “Guides” section, separate from the examples.

**Comment by 1ec5** [comment]
There’s no reason why we can’t have both guides and examples:

```
--documentation=docs/{guides,examples}/*.md
```

File: docs/jazzy.yml [hunk 2]
```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
...
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
+      - Advanced
+      - Basic
+      - Embedded-Navigation
+      - Styled-UI-Elements
+      - Waypoint-Arrival-Screen
   - name: Guides
```
Comment (1ec5) [inline docs/jazzy.yml]: Probably makes sense to move “Guides” up above “Examples”. Eventually we’d add more conceptual content here that could serve as a prerequisite for the examples.

File: docs/jazzy.yml [hunk 3]
```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
...
 
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
+      - Advanced
+      - Basic
+      - Embedded-Navigation
+      - Styled-UI-Elements
+      - Waypoint-Arrival-Screen
```
Comment (1ec5) [inline docs/jazzy.yml]: This looks really weird. The script should replace hyphens with spaces in the file names.

File: docs/examples/StyledUIElements.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
+
+_Source available [here](https://github.com/mapbox/navigation-ios-examples/blob/master/Navigation%20Examples/Examples/StyledUIElements.swift)_
```
Comment (1ec5) [inline docs/examples/StyledUIElements.md]: Both Styled-UI-Elements.md and StyledUIElements.md?

File: docs/jazzy.yml [hunk 5]
```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
 hide_documentation_coverage: Yes
 
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
```
Comment (1ec5) [inline docs/jazzy.yml]: “Storyboards” is already listed under “Guides”.

File: docs/jazzy.yml [hunk 6]
```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
...
+      - Advanced
+      - Basic
```
Comment (1ec5) [inline docs/jazzy.yml]: Rearrange these examples in order of difficulty, so that “Basic” comes before “Advanced”.

**Comment by bsudekum** [comment]
@1ec5 fixed

**Pull request merged**